frontend/model/contracts/group.js

```diff
@@ -99,44 +99,6 @@ function initFetchMonthlyPayments ({ meta, state, getters }) {
   return monthlyPayments
 }
 
-function groupIncomeDistribution ({ state, getters, monthstamp, adjusted }) {
```

Comment:
note: this function was moved to `utils/distribution/group-income-distribution.js`